### ISSUE TITLE
Implement standardized in app authentication process

### ISSUE THREAD

**Issue by J-Son89** [status-im/status-react#15570]
Design team are currently working on a standardised authentication process for in app use. 

Designs: 
https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?node-id=1104-155973&t=CjF08I1JH0ASv6JA-0

https://www.figma.com/file/zG04IdiVCb4ZNC5n3nH9yg/Authentication-for-Mobile?type=design&node-id=0%3A1&t=nnuEZbGYPpwGAGcW-1

The solution should also work for these pages- https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1741-374592&mode=design&t=FGZZnmf1pJQzj0d7-0
 i.e it is a generic approach and has the required callbacks for success/failure states etc 

Double check with design team these are ready to go before starting 👍 

**Keycard variant is out of scope!**

**Comment by J-Son89** [comment]
cc @pedro-et @xAlisher

**Comment by flexsurfer** [comment]
https://www.figma.com/file/zG04IdiVCb4ZNC5n3nH9yg/Authentication-for-Mobile?type=design&node-id=0%3A1&t=nnuEZbGYPpwGAGcW-1

**Comment by hysekbutt** [comment]
Hi @pedro-et,
I'm trying to open the following figma designs, but I'm unable to access these, seems like I don't have access to these. Can you please give me access to these designs so that I can start working on this task.

https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?node-id=1104-155973&t=CjF08I1JH0ASv6JA-0

https://www.figma.com/file/zG04IdiVCb4ZNC5n3nH9yg/Authentication-for-Mobile?type=design&node-id=0%3A1&t=nnuEZbGYPpwGAGcW-1

https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1741-374592&mode=design&t=FGZZnmf1pJQzj0d7-0

**Comment by flexsurfer** [comment]
сс @Francesca-G

**Comment by Francesca-G** [comment]
> Hi @pedro-et, I'm trying to open the following figma designs, but I'm unable to access these, seems like I don't have access to these. Can you please give me access to these designs so that I can start working on this task.
> 
> https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?node-id=1104-155973&t=CjF08I1JH0ASv6JA-0
> 
> https://www.figma.com/file/zG04IdiVCb4ZNC5n3nH9yg/Authentication-for-Mobile?type=design&node-id=0%3A1&t=nnuEZbGYPpwGAGcW-1
> 
> https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1741-374592&mode=design&t=FGZZnmf1pJQzj0d7-0

I will need your email address to add you as a guest :)
